src/acquisition/covidcast/csv_importer.py

```diff
@@ -271,13 +279,62 @@ def extract_and_check_row(row, geo_type):
 
     # optional not-too-small float
     try:
+      # Returns either a float, None, or causes an exception
       sample_size = CsvImporter.maybe_apply(float, row.sample_size)
     except ValueError:
       # expected a number, but got a string
       return (None, 'sample_size')
 
+    if hasattr(row, "missing_val"):
+      missing_value = row.missing_val
+      # not an int code
+      if not isinstance(missing_value, int):
+        # capture a few possible ways to write a nan
+        if str(missing_value).lower() in {'na', 'nan', 'none', ''}:
+          missing_value = 0
+        # if not an int or a nan, then we have problems
+        else:
+          return (None, 'missing_val')
+      # missing code contradicts the value being present
+      elif value is not None and missing_value > 0:
+        return (None, 'missing_val')
+    else:
+      missing_value = 0
+
+    if hasattr(row, "missing_se"):
```

Comment:
we should standardize on se vs stderr

Reply:
yea, so in the CSV files, the standard is `val`, `se`, and `sample_size`. in the database, these get expanded out to `value`, `stderr`, and `sample_size`. the attributes of `RowValues` reflects that. we could change the database to make it more consistent. there would probably be many downstream SQL queries that would need to change then.

Reply:
I see, didn't realize the db had the stderr for se already. agree that missing_stderr should be consistent